Cargo.toml

```
@@ -7,9 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-approx = "0.4.0"
+approx = "0.5"
```

should this be patched version?

```suggestion
approx = "0.5.0"
```